src/main/java/com/byznass/tiolktrack/kernel/dao/UserProvider.java

```diff
@@ -0,0 +1,8 @@
+package com.byznass.tiolktrack.kernel.dao;
+
+import com.byznass.tiolktrack.kernel.model.User;
+
+public interface UserProvider {
+
+	User getUser(String userId) throws NoUserWithSuchIdException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.byznass.tiolktrack.kernel.dao.NoUserWithSuchIdException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)